src/cli/opts.rs

```diff
@@ -58,24 +58,24 @@ pub enum Command {
     #[clap(aliases = &["ls"])]
     ListSwaps,
 
-    /// Lists public offers created by daemon
+    /// Lists public deals created by daemon
     #[clap(aliases = &["lo"])]
```

Comment:
Change to `ld` aliases

Reply:
64b2f5c7fd5ce32f6b166ed1a6f8f9dfb62f2606